.changeset/witty-cats-tell.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': patch
```

Comment:
Don't need this `example-app` change in here as we don't publish that package to npm